docs/source/reference/utils.rst

```diff
@@ -3,6 +3,23 @@
 Utilities
 =========
 
+Printing
+========
+
+
+.. function:: numba.misc.utils.printf(format, ...)
+
+   Writes the C string pointed by `format` to the standart output (stdout).
+   If `format` includes specifiers, the additional arguments following format
+   are inserted in the resulting string. The following formatters are supported:
+   `%d`, `%i`, `%u`, `%x`, `%X`, `%f`, `%F`, `%e`, `%E`, `%g`, `%G`, `%c`, and `%s`
```

Comment:
no reference to %o or %p even though there tests for them.

Reply:
Fixed